Documentation/guides/building-apps/build-properties.md

```diff
@@ -908,6 +908,32 @@ properties are set, which are required for Android App Bundles:
 [apk]: https://en.wikipedia.org/wiki/Android_application_package
 [bundle]: https://developer.android.com/platform/technology/app-bundle
 
+This property will be deprecated for .net 6. Users should switch over to
+the newer [`AndroidPackageFormats`](~/android/deploy-test/building-apps/build-properties.md#androidpackageformats).
+
+## AndroidPackageFormats
+
+A semi-colon delimited property with valid values of `apk` and `aab`.
+This indicates if you want to package the Android application as
+an [APK file][apk] or [Android App Bundle][bundle]. App Bundles
+are a new format for `Release` builds that are intended for
+submission on Google Play.
+
+When building a Release build you might want to generate both
+and `aab` and an `apk` for distribution to various stores.
+
+Setting `AndroidPackageFormats` to `aab;apk` will result in both
+being generated. Setting `AndroidPackageFormats` to either `aab`
+or `apk` will generate only one file.
+
+For .net 6 `AndroidPackageFormats` will be set to `aab;apk` for
+`Release` builds only. It is recommended that you continue to use
+just `apk` for debugging.
+
+For Legacy Xamarin.Android this value currently defaults to `apk`.
+
+Added in Xamarin.Android 11.5.
```

Comment:
@jonathanpeppers: What do we call the final .NET 6 Android for .NET (or whatever) version?  Assuming that this new property is only supported in .NET6+, what should we say here?

Reply:
I think we should put `.NET 6 Preview 7`, if this is where the fix would land.

This currently is working in legacy as well, though, right? I don't see what it would hurt if it was added to legacy, as it's just an extra target that runs for a new MSBuild property that didn't exist before.

Reply:
@jonpryor is 11.5 the right version? I can never remember where to look for the next release number.
Also do we need to start putting in the .net 6 version as well or instead of?

Reply:
I think 11.5 is right because main is:

https://github.com/xamarin/xamarin-android/blob/21751b5bc4292be4b23dd58577cb6cf9f01d82bb/Directory.Build.props#L16